src/webserver/Dynamic.jl

```diff
@@ -106,6 +106,9 @@ function notebook_to_js(notebook::Notebook)
                 "queued" => cell.queued,
                 "running" => cell.running,
                 "errored" => cell.errored,
```

Comment:
`notebook_to_js` function takes our cell variables and throws it to the other side